app/client/views/activities/activities.js

```diff
@@ -12,23 +12,36 @@ Template.activities.onCreated(function() {
 
   instance.currentPageOfActivities = new ReactiveVar([]);
   instance.rowsPerPage = new ReactiveVar(10);
+  instance.currentPage = new ReactiveVar(1);
   instance.totalRows = new ReactiveVar(0);
-  instance.reset = new ReactiveVar(true);
+
+  this.autorun(function() {
+    const activityDeleted = Session.get('activity-deleted');
+    const activityEdited = Session.get('activity-edited');
+    const currentPage = instance.currentPage.get();
+    const rowsPerPage = instance.rowsPerPage.get();
+    if (activityDeleted === true || activityEdited===true) {
+      onChange(instance, rowsPerPage, currentPage);
```

Comment:
Just add some descriptive comments here, as this is the most important code. In particular, describe:

- what `onChange` is doing, as if you were teaching a junior developer or writing a tutorial
- why we needed to use `Session` here, as it is somewhat unconventional in this project

Reply:
Done